src/utils/project-content/get-project-content.ts

```diff
@@ -194,32 +194,47 @@ function _getCollectionFragmentCompositions(
 }
 
 function _getPageTemplates(basePath: string): IPageTemplate[] {
+  const displayPageTemplates = _getPageTemplatesByType(basePath, 'display-page-template');
+  const pageTemplates = _getPageTemplatesByType(basePath, 'page-template');
+  const masterPages = _getPageTemplatesByType(basePath, 'master-page');
+
+  return Array.of(
+     ...displayPageTemplates,
+     ...pageTemplates,
+     ...masterPages
+  );
+}
+
+function _getPageTemplatesByType(basePath: string, type: 'display-page-template' | 'page-template' | 'master-page'): IPageTemplate[] {
```

Comment:
I think we should extract this to a type declaration [here](https://github.com/liferay/generator-liferay-fragments/pull/193/commits/9ae76a77f5bce3709f2eaf37dd8b0b47418e1764#diff-093ad82a25aee498b11febf1cdcb6546e4d223ffcb49ed69cc275ac27ce0ccceR95) to avoid duplication :3